docs/theme-specification.md

```diff
@@ -131,68 +160,55 @@ m: [ 0, 1, 2 ]
 }
 ```
 
-#### Media Queries
-
-For convenience and for use with other styling approaches, a `mediaQueries` scale derived from the `breakpoints` scale can be added to the theme object.
-
-```js
-breakpoints: [ '40em', '52em', '64em' ]
-
-mediaQueries: {
-  small: `@media screen and (min-width: ${breakpoints[0]})`,
-  medium: `@media screen and (min-width: ${breakpoints[1]})`,
-  large: `@media screen and (min-width: ${breakpoints[2]})`,
-}
-```
-
-### Key Reference
+### Key Reference / aliases
 
 The following is a list of theme object keys and their corresponding CSS properties.
 This list may be non-exhaustive.
 
 Theme Key         | CSS Properties
 ------------------|--------------
 `space`           | `margin`, `margin-top`, `margin-right`, `margin-bottom`, `margin-left`, `padding`, `padding-top`, `padding-right`, `padding-bottom`, `padding-left`, `grid-gap`, `grid-column-gap`, `grid-row-gap`
-`fontSizes`       | `font-size`
-`colors`          | `color`, `background-color`, `border-color`
-`fonts`           | `font-family`
-`fontWeights`     | `font-weight`
-`lineHeights`     | `line-height`
-`letterSpacings`  | `letter-spacing`
-`maxWidths`       | `max-width`
-`minWidths`       | `min-widths`
-`widths`          | `width`
-`maxHeights`      | `max-height`
-`minHeights`      | `min-height`
-`heights`         | `height`
-`borders`         | `border`, `border-top`, `border-right`, `border-bottom`, `border-left`
-`borderWidths`    | `border-width`
-`borderStyles`    | `border-style`
+`font.size`       | `font-size`
+`font.family`     | `font-family`
+`font.weight`     | `font-weight`
+`lineHeight`      | `line-height`
+`color`           | `color`, `background-color`, `border-color`
+`letterSpacing`   | `letter-spacing`
+`maxWidth`        | `max-width`
+`minWidth`        | `min-widths`
+`width`           | `width`
+`maxHeight`       | `max-height`
+`minHeight `      | `min-height`
+`height`          | `height`
+`border`          | `border`, `border-top`, `border-right`, `border-bottom`, `border-left`
+`border.width`    | `border-width`
 `radii`           | `border-radius`
-`shadows`         | `box-shadow`, `text-shadow`
-`zIndices`        | `z-index`
+`boxShadow`       | `box-shadow`, `text-shadow`
```

Comment:
Would you propose a separate `textShadow` field with this direction?

Reply:
that'd make sense if we go for the singular form and css properties names. Otherwise I think the your original idea is great:

Reply:
I personally prefer the plural form since they often map to a collection of values, but it's definitely not a hill I would die on so long as we're consistent.